asv_bench/benchmarks/series_methods.py

```diff
@@ -25,7 +25,7 @@ def time_constructor(self, data):
 
 class IsIn:
 
-    params = ["int64", "uint64", "object"]
+    params = ["int64", "uint64", "object", "Int64"]
     param_names = ["dtype"]
 
     def setup(self, dtype):
```

Comment:
can you make the size a parameter in the class, e.g. N and make it 10000 is fine

Reply:
Done.